Documentation/project-docs/triage.md

```diff
@@ -48,4 +48,25 @@ In particular, we strive to follow these guidelines:
 * **Do** close issues that represent work we don't think makes sense for the current or upcoming release
 * **Do** close issues that provide little or questionable value to APIs that are included with the .NET Framework
 * **Do** favor issues that do not require modifying types that are included with the .NET Framework.
-* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
\ No newline at end of file
+* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
+
+## Porting to .NET Framework
+
+### Constraints for shipping in the .NET Framework
+ 
+Each release of .NET Framework ships to over one billion machines word-wide and installs as an *in-place* update, requiring each fix to meet a high level of quality and compatibility.  Each fix is reviewed extensively to determine which release if any is most appropriate.
```

Comment:
Our doc doesn't plan that far ahead. We're covering the next 6 to 12 months, and yes, in that time frame there are no plans to accommodate compat risks by shipping a side-by-side version of the .NET Framework.